src/drawerContainer/drawerContainer.component.js

```diff
@@ -86,6 +72,9 @@ const mapStateToProps = state => ({
   open: state.viewState.openParentPanel,
   openSecondary: state.malcolm.childBlock !== undefined,
   urlPath: state.router.location.pathname,
+  baseUrl: Object.prototype.hasOwnProperty.call(state, 'navigation')
```

Comment:
I'm not sure there's a `state.navigation` property, any idea what this is meant to be?

Reply:
we've established in a future commit we'll use `window.location.host`